microsoft-edge/web-platform/os-regional-settings.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Updates to Region information exposed to web content
+description: How customers and web developers can use the OS regional format in Microsoft Edge for improved site experiences
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 08/11/2022
+--- 
+# Updates to Region information exposed to web content
```

Comment:
better title possible, relating to user POV?  Is the article about 1 specific new UI control/setting that could be highlighted in the title?  If so, change TOC.yml & "title: " YAML field above & this h1.

Reply:
> Is the article about 1 specific new UI control/setting that could be highlighted in the title?

Yes, will suggest a better title...

Reply:
done